src/cloudai/_core/runner.py

```diff
@@ -43,7 +43,6 @@ class Runner:
     _runners = {}
 
     def __init__(self, mode: str, system: System, test_scenario: TestScenario):
-        logging.info("Initializing Runner")
```

Comment:
Why moving this log line from here? IMO it make sense to keep it here as the most high-level part of runners logic.

Reply:
Okay, then I will update this line instead.